Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -292,9 +249,15 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
     /// - Parameter dataProvider: The data provider to register bundles from.
     /// - Parameter diagnosticEngine: The pre-configured engine that will collect problems encountered during compilation.
     /// - Throws: If an error is encountered while registering a documentation bundle.
-    public init(dataProvider: DocumentationContextDataProvider, diagnosticEngine: DiagnosticEngine = .init()) throws {
+    public init(
+        dataProvider: DocumentationContextDataProvider,
+        diagnosticEngine: DiagnosticEngine = .init(),
+        configuration: Configuration = .init()
```

Comment:
I forgot to document this parameter

Reply:
Resolved in https://github.com/swiftlang/swift-docc/pull/1031/commits/13575915e60cefbe6b7974bf90bfdd34ef31ce5b